lib/core/src/lib/form/components/widgets/core/form-field.model.ts

```diff
@@ -306,29 +307,34 @@ export class FormFieldModel extends FormWidgetModel {
         /*
          This is needed due to Activiti issue related to reading dropdown values as value string
          but saving back as object: { id: <id>, name: <name> }
+         Side note: Probably not valid anymore
```

Comment:
what is a comment of a comment? please remove it if not valid.

Reply:
Still not 100% sure that we can remove this comment, but indeed I asked a lots of people and they can't recall this drawback and in runtime we can't observe this behavior as far as I know.